XV/Assets/Scripts/GameManager.cs

```diff
@@ -193,10 +193,13 @@ public GameObject BuildObject(ObjectDataScene iODS, bool iAnimatedPopping = fals
 
 
 		// If this item can move - Add MovableEntity script
-		if (lParameters != null && lParameters.Movable) {
-			oGameObject.AddComponent<MovableEntity>()
-					   .SetEntity(lObjectEntity);
-		}
+		if (lParameters != null) {
+            if (lParameters.Movable) {
+                oGameObject.AddComponent<MovableEntity>()
+                           .SetEntity(lObjectEntity);
+            } else
+                lUIBubbleInfo.GetComponent<UIBubbleInfo>().HideSpeedInput();
```

Comment:
You charge again GameManager while you could do it in a PostPoppingAction or in the UIBubbleInfo.start()